internal/worker/api/openapi.yml

```diff
@@ -203,6 +204,43 @@ paths:
               schema:
                 $ref: '#/components/schemas/Error'
 
+  /workers:
+    post:
+      operationId: postWorkers
+      summary: Create a new worker
```

Comment:
I found this weird too and had the exact same thought (not a REST API designer, so maybe there are conventions I'm not aware of).  Maybe naming the endpoints `worker` would imply that only one worker is interacting?

Probably not that important for the worker API anyway.

Reply:
Thought it was a rest convention to use the plural.

Reply:
Based on an off-channel discussion, this really is a convention.

Reply:
indeed, it's convention to use plural for items which are collections, since there are multiple workers (we just don't have a listing endpoint), the endpoint should be plural.